applets/clock/clock.c

```diff
@@ -682,6 +773,64 @@ update_tooltip (ClockData * cd)
                 else
                         tip = _("Click to view month calendar");
         }
+        if (!cd->showdate) {
```

Comment:
You don't need to show date and time on the tooltip, furthermore tip was set on the previous line.